src/components/managersinglecomm/ManagerSingleCommunity.js

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image'
```

Comment:
> @TaylorDJones11 I think instead of `managersinglecomm` in the file path, we can use the simpler `dashboard` what do you think?

I was only doing the Single Community Component while @istoga has the entire dashboard. I can change the name but I didn't want to get it mixed up with Neka's possibly name

Reply:
This is alright; we can fix any clashes during merge conflict resolution.
I think it makes sense for all `dashboard`-related components to live inside `dashboard` - what do you think @istoga?

Reply:
> This is alright; we can fix any clashes during merge conflict resolution. I think it makes sense for all `dashboard`-related components to live inside `dashboard` - what do you think @istoga?

I've changed the component folders/function/styling to match with "Dashboard" throughout